lib/ansible/plugins/connection/winrm.py

```diff
@@ -193,6 +195,13 @@ def __init__(self, *args, **kwargs):
 
         super(Connection, self).__init__(*args, **kwargs)
 
+        if C.DEFAULT_DEBUG:
+            logging.getLogger('requests_credssp').setLevel(logging.DEBUG)
+            logging.getLogger('urllib3').setLevel(logging.DEBUG)
+        else:
+            logging.getLogger('requests_credssp').setLevel(logging.WARNING)
```

Comment:
Set to `INFO` as well as urlib3 below. We should also add `requests_kerberos` as that has a few .debug log statements.

Reply:
Ok.

Reply:
Fixed.